### ISSUE TITLE
Better HTTP Response

### ISSUE THREAD

**Issue by shirshak55** [clifinger/canduma#17]
Currently response are not all json. For example if we register using same information twice following is the raw data instead of helpful json message like Wrong username or password.

```
BadRequest: Key (email)=(shirshak55@gmail.com) already exists.
```

**Comment by clifinger** [comment]
Correct, I will work on it ... 
Bad week-end, I have flu :(

**Comment by clifinger** [comment]
Resolved in #19

**Comment by shirshak55** [comment]
great
